sidebars.js

```diff
@@ -386,6 +386,10 @@ module.exports = {
                             "type": "doc",
                             "id": "learn/bossroom/spawn-networkobjects"
                         },
+                        {
+                            "type": "doc",
+                            "id": "learn/porting-to-dgs"
```

Comment:
It's a bit weird this appears under Boss Room when BR doesn't have DGS mode. BR was _used_ for research while writing this doc, but it doesn't contain any examples for it.
This is a mix bag. It contains API specific things, but also generic advice on tech in general.
This could go under Networking under Relay?
This could also go in Concepts and FAQs under Multiplayer Game Architecture?
I think it makes more sense to put this after Relay.
But anyway, we have a meeting to reorganize that side bar next year, this is most likely to change anyway.
All in all, I would still not put it under boss room, there's no link between the two, not right now.

Reply:
Yeah, I wasn't sure about putting it under Boss Room. My thinking was that we're going to reorganize the sidebar next year anyway. But I agree - after Relay would probably be best. Thanks!